config/locales/javascript/javascript.en.yml

```diff
@@ -297,7 +297,12 @@ en:
       original_post: "original post"
       result: "Result"
       count:
+        zero: "No vote so far"
         one: "1 vote so far"
         other: "<%=count%> votes so far"
+      answer_count:
+        zero: "0 vote"
```

Comment:
But it will still be presented to translators in webtranslateit automatically right?
Good then, I'll revert that.